enclave-manager/web/src/components/enclaves/widgets/DownloadFileButton.tsx

```diff
@@ -0,0 +1,45 @@
+import { useToast } from "@chakra-ui/react";
+import { FilesArtifactNameAndUuid } from "enclave-manager-sdk/build/api_container_service_pb";
+import { useState } from "react";
+import { useKurtosisClient } from "../../../client/enclaveManager/KurtosisClientContext";
+import { EnclaveFullInfo } from "../../../emui/enclaves/types";
+import { saveTextAsFile } from "../../../utils/download";
+import { DownloadButton } from "../../DownloadButton";
+
+type DownloadFileButtonProps = {
+  file: FilesArtifactNameAndUuid;
+  enclave: EnclaveFullInfo;
+};
+
+export const DownloadFileButton = ({ file, enclave }: DownloadFileButtonProps) => {
+  const kurtosisClient = useKurtosisClient();
+  const toast = useToast();
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleDownloadClick = async () => {
+    setIsLoading(true);
+    // todo: get tgz download instead
```

Comment:
would it be here Ben?

Reply:
yah, needs to be wired through from `KurtosisClient`